apps.json

```diff
@@ -564,6 +564,15 @@
         ],
         "url": "https://github.com/YunoHost-Apps/dendrite_ynh"
     },
+    "dex": {
+        "category": "system_tools",
+        "maintained": true,
+        "state": "inprogress",
+        "subtags": [
+            "auth"
```

Comment:
I don't know if we are allowed to add this subtag... (I can't see it in the system tools subtags section)